ports/libnpy/portfile.cmake

```
@@ -0,0 +1,25 @@
+vcpkg_from_github(

```

```suggestion
vcpkg_check_linkage(ONLY_STATIC_LIBRARY)

vcpkg_from_github(
```
Upstream only provides a static build.